### PR TITLE
Fixing bug in PTVS for UWP python scenarios

### PR DESCRIPTION
**Pull request by alanch-ms**
- Update PTVS version number and directory location
- Fixed an issue where the VS cannot attach the python debugger due to a malformated tcp string.

@zooba - For bug fixes in update 1?  Do I commit them in Master or v2.2.1 branch?  The first commit (Update PTVS version number and directory location) probably doesn't need to go into the v2.2.1 branch.

### CODE REVIEW

File: Python/Product/Uwp/Project/PythonUwpProjectConfig.cs [hunk 1]
```diff
@@ -100,6 +100,13 @@ class PythonUwpProjectConfig :
             const int attachRetryLimit = 10;
             int attachRetryCount = 0;
 
+            // Remove the port number if exist
+            int index = remoteMachine.IndexOf(':');
+            if (index != -1)
+            {
```
Comment (zooba) [inline Python/Product/Uwp/Project/PythonUwpProjectConfig.cs]: We use same-line opening brace style consistently.

**Comment by zooba** [comment]
For update one they'll need to go on v2.2.1, but since you've done separate commits I can easily backport it, so don't worry too much.

**Comment by alanch-ms** [comment]
Fixed the "same-line opening brace" issue.

**Comment by zooba** [comment]
I'm working on our CI today, so I'm going to hold off on merging until I want to trigger it. But this will go in for v2.2.1 (not the RC though).

**Pull request merged**